integrated_channels/blackboard/models.py

```diff
@@ -88,3 +94,93 @@ def get_content_metadata_exporter(self, user):
 
     def get_content_metadata_transmitter(self):
         return BlackboardContentMetadataTransmitter(self)
+
+    def get_learner_data_exporter(self, user):
+        return BlackboardLearnerExporter(user, self)
+
+    def get_learner_data_transmitter(self):
+        return BlackboardLearnerTransmitter(self)
+
+
+@python_2_unicode_compatible
+class BlackboardLearnerDataTransmissionAudit(models.Model):
+    """
+    The payload we send to Blackboard at a given point in time for an enterprise course enrollment.
+
+    """
+    blackboard_user_email = models.EmailField(
+        max_length=255,
+        blank=False,
+        null=False,
+        help_text='The learner`s Blackboard email. This must match the email on edX'
```

Comment:
Love the comment! maybe add a suffix like ' must match the email on edX in order to ...' ? technically blackboard will take any email but the issue will be during SSO back to edX... not sure if you want to say all that here lol